OakCityV2/TableViewController.h

```diff
@@ -10,6 +10,6 @@
 
 @interface TableViewController : UITableViewController <UIAlertViewDelegate>
 
-@property (strong, nonatomic) NSArray *jobs;
+@property (strong, nonatomic) NSMutableArray *jobs;
```

Comment:
So this is subtle.  Having a mutable array means you can change the array's contents.  Having an immutable array means you can't change the array's contents, but you can swap out a whole array.  Since you never intend to change the contents of the array once it's built, you should keep this as an immutable NSArray, not an NSMutableArray.